packages/ui/components/calendar/src/utils/dataTemplate.js

```diff
@@ -13,23 +13,12 @@ export function dataTemplate(
     <div id="js-content-wrapper">
       ${data.months.map(
         month => html`
-          <table
-            role="grid"
-            data-wrap-cols
-            aria-readonly="true"
-            class="calendar__grid"
-            aria-labelledby="month year"
-          >
+          <table role="grid" data-wrap-cols aria-readonly="true" class="calendar__grid">
             <thead>
-              <tr role="row">
```

Comment:
Note we should keep the roles. I think this discussion pops up every now and then again :)

The reason is: lion is made witg extending it in mind. That means overriding css. 
If you change the display prop of tables, it loses its semantics

Reply:
I didn't know about this one, oke will rollback that part and add a note somewhere

Reply:
Do we also have a rationale about other changes? Like removing aria-labelledby and removing weekday names?

Reply:
If its duplicate? How exactly? And for which screen readers?

Reply:
Let me list them in the description of the PR.